src/Microsoft.Data.SqlClient/netcore/src/Resources/Strings.Designer.cs

```diff
@@ -3525,6 +3525,15 @@ internal class Strings {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to The DataAdapter for which to automatically generate SqlCommands..
+        /// </summary>
+        internal static string SqlCommandBuilder_DataAdapter {
+            get {
+                return ResourceManager.GetString("SqlCommandBuilder_DataAdapter", resourceCulture);
+            }
+        }
+        
```

Comment:
We don't include the resources like this in `netcore` which are just used in attributes; They're supposed to be merged in the future and we have a temporary place for them. You can add a constant for it in [StringsHelper.ResourceNames](https://github.com/lcheunglci/SqlClient/blob/MergeShared-SqlCommandBuilder/src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs#L75). So, you can revert this change.